src/components/Avatar/__stories__/Avatar.stories.tsx

```diff
@@ -0,0 +1,285 @@
+
+const randomAvatars = faker.helpers
+    .uniqueArray(() => faker.number.int({min: 20, max: 500}), 30)
+    .reduce(
+        (sizes, num) => ({
+            ...sizes,
+            [num]: faker.image.urlLoremFlickr({
+                category: 'cats',
+                width: num,
+                height: Math.round((num / 640) * 480),
+            }),
+        }),
+        {},
+    );
+
+const imageProps = {
+    imgUrl,
+};
+
+const iconProps = {
+    backgroundColor: 'var(--g-color-base-brand)',
+    icon: FaceRobot,
+    color: 'var(--g-color-text-brand-contrast)',
+};
+
+const textProps = {
+    backgroundColor: 'var(--g-color-base-generic-medium)',
+    text: 'Charles Darwin',
+    color: 'var(--g-color-text-primary)',
+};
+
+const BORDER_COLOR = 'var(--g-color-line-misc)';
+
+export const Image: Story = {
+    args: {
+        imgUrl,
+    },
+};
+
+export const ImageSrcSet: StoryFunc = (args) => {
```

Comment:
Let's use StoryObj format here as well, via `{render: () => {}}`

Reply:
The linter gives an error, that you can't use hooks outside component